src/jquery.data-remote.js

```diff
@@ -30,6 +30,7 @@
       before: function($target) {} // callback fires directly before the request is made
     };
 
+    var debouncableEvents = ['keyup', 'keydown', 'keypress', 'resize', 'resize'];
```

Comment:
not a huge deal but would also rather `options` went right below `defaults` so can you reverse the order of `options` and `debouncableEvents`?